### ISSUE TITLE
No invitation link with discord connector

### ISSUE THREAD

**Issue by xavier-rodet** [csvance/FTBot#52]
I built tensorflow (r2.0) from source to add AVX2 FMA cpu support (and because basic pip installation wasn't working either).

 Here what i got when i start `python armchair_expert.py --retrain-structure` (running on docker Ubuntu 18.04) :

> INFO:ArmchairExpert:Status: STARTING_UP
> /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_qint8 = np.dtype([("qint8", np.int8, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_qint16 = np.dtype([("qint16", np.int16, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_qint32 = np.dtype([("qint32", np.int32, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   np_resource = np.dtype([("resource", np.ubyte, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_qint8 = np.dtype([("qint8", np.int8, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_qint16 = np.dtype([("qint16", np.int16, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   _np_qint32 = np.dtype([("qint32", np.int32, 1)])
> /usr/local/lib/python3.6/dist-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
>   np_resource = np.dtype([("resource", np.ubyte, 1)])
> Using TensorFlow backend.
> WARNING: Logging before flag parsing goes to stderr.
> W0805 14:12:44.053263 139799969265472 deprecation_wrapper.py:118] From /usr/local/lib/python3.6/dist-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.
> 
> W0805 14:12:44.061141 139799969265472 deprecation_wrapper.py:118] From /usr/local/lib/python3.6/dist-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.
> 
> W0805 14:12:44.062849 139799969265472 deprecation_wrapper.py:118] From /usr/local/lib/python3.6/dist-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.
> 
> W0805 14:12:44.131823 139799969265472 deprecation_wrapper.py:118] From /usr/local/lib/python3.6/dist-packages/keras/backend/tensorflow_backend.py:133: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.
> 
> W0805 14:12:44.137647 139799969265472 deprecation.py:506] From /usr/local/lib/python3.6/dist-packages/keras/backend/tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
> Instructions for updating:
> Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
> W0805 14:12:44.326126 139799969265472 deprecation_wrapper.py:118] From /usr/local/lib/python3.6/dist-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.
> 
> W0805 14:12:44.340184 139799969265472 deprecation_wrapper.py:118] From /usr/local/lib/python3.6/dist-packages/keras/backend/tensorflow_backend.py:3341: The name tf.log is deprecated. Please use tf.math.log instead.
> 
> 2019-08-05 14:12:44.352372: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3795880000 Hz
> 2019-08-05 14:12:44.352505: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x1cbc580 executing computations on platform Host. Devices:
> 2019-08-05 14:12:44.352545: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
>
